backend/config/curation-api.yml

```diff
@@ -159,10 +175,12 @@ paths:
           $ref: "#/components/responses/403"
     patch:
       summary: Update a Collection's metadata.
-      description: >-
+      description: |
         Update a private Collection or a Revision. If any Collection metadata fields are included in the body of the
-        request, these fields SHALL be updated for the specified Collection. If links are included, they will fully
-        replace all existing links. A DOI link that cannot be found at Crossref results in a 400 (BAD REQUEST).
+        request, these fields will be updated for the specified Collection. If links are included, they will fully
+        replace all existing links. CELLxGENE Discover queries **Crossref** (link to their site) for the publication 
```

Comment:
replace placeholder with link to Crossref site

Reply:
I'll fix this in a different PR